app/assets/locale/en.json

```diff
@@ -308,7 +308,7 @@
     }
   },
   "validateSurname": {
-    "heading": "Enter your last name",
+    "heading": "Enter your last name - Track your benefit appeal - GOV.UK",
```

Comment:
Should this be Appeal a benefit decision as well?

Reply:
Oh yes! well spotted.